Library/Formula/libgpm.rb

```
@@ -0,0 +1,58 @@
+class Libgpm < Formula
+  desc "general purpose mouse"
+  homepage "http://www.nico.schottelius.org/software/gpm/"
+  url "http://www.nico.schottelius.org/software/gpm/archives/gpm-1.20.7.tar.bz2"
+  sha256 "f011b7dc7afb824e0a017b89b7300514e772853ece7fc4ee640310889411a48d"
+
+  head "https://github.com/telmich/gpm.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "bison" => :build
+  depends_on "texinfo" => :build
+  depends_on "homebrew/dupes/ncurses" unless OS.mac?
+
+  # https://github.com/telmich/gpm/pull/14
+  patch :DATA
+
+  def install
+    ENV.deparallelize
```

Please report the error message that you see without `ENV.deparallelize`